README.md

```diff
@@ -42,22 +42,27 @@ Read more at [hasura.io](https://hasura.io) and the [docs](https://docs.hasura.i
 <!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
 **Table of Contents**
 
-- [Quickstart:](#quickstart)
```

Comment:
I can still see some level of nesting in the table of contents. I think everything from line 45 to line 95 should be reverted to the original. Can you do that? The rest looks good 🙂